containers/dotnet/definition-manifest.json

```diff
@@ -7,6 +7,14 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
+		"architectures": {
+			"6.0-focal": ["linux/amd64", "linux/arm64"],
+			"6.0-bullseye-slim": ["linux/amd64", "linux/arm64"],
+			"5.0-focal": ["linux/amd64", "linux/arm64"],
+			"5.0-bullseye-slim": ["linux/amd64", "linux/arm64"],
+			"3.1-focal": ["linux/amd64", "linux/arm64"],
+			"3.1-bullseye": ["linux/amd64", "linux/arm64"]
+		},
 		"variantTags": {
 			"6.0-bullseye-slim": [
```

Comment:
<low-pro> I realize is unrelated to these changes -- I think the `variantTags` section should also include "6.0-focal", based on what I see [published here](https://mcr.microsoft.com/v2/vscode/devcontainers/dotnet/tags/list) :

			"6.0-focal": [ 
				"dotnet:${VERSION}-6.0",
				"dotnetcore:${VERSION}-6.0"
			],

Reply:
@dnastrain, good catch.

Reply:
updated!